here-naksha-lib-model/src/commonMain/kotlin/naksha/model/request/SuccessResponse.kt

```diff
@@ -30,6 +31,11 @@ open class SuccessResponse @Deprecated("Use secondary", ReplaceWith("SuccessResp
         this.features.addAll(tuples.map { it.feature })
     }
 
+    @JsName("ofFeatures")
+    constructor(features: NakshaFeatureList) : this() {
```

Comment:
Unfortunately not, building it will cause this error
```
Platform declaration clash: The following declarations have the same JVM signature (<init>(Ljava/util/List;)V):
    constructor SuccessResponse(features: List<NakshaFeature?>) defined in naksha.model.request.SuccessResponse
    constructor SuccessResponse(tuples: List<ResultTuple>) defined in naksha.model.request.SuccessResponse
```
Apparently both constructor params will be compiled to `List<?>` in java bytecode